### PR TITLE
updated powerbi-api to version 5.4.0

### PR DESCRIPTION
**Pull request by SanderGi**
## What did I do?
I updated the powerbi-visuals-api package to version 5.4.0 in packages.json and the pbiviz.json config file accordingly

## Why did I do it this way?
Update the visuals-api to allow adding a formatting pane in a late PR

## Anything else you should know?
Linting doesn't pass but that is unrelated to my changes (did not pass before). I have another PR that fixes this

## Screenshots
No visual changes, only changed package version
image

### CODE REVIEW

**Comment by SanderGi** [comment]
@microsoft-github-policy-service agree

On Jun 26, 2023, at 5:01 PM, microsoft-github-policy-service[bot] ***@***.******@***.***>> wrote:


@microsoft-github-policy-service agree [company="{your company}"]


Options:

  *   (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.

@microsoft-github-policy-service agree


  *   (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.

@microsoft-github-policy-service agree company="Microsoft"

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "less-loader": "^7.2.1",
     "lodash-es": "4.17.20",
     "powerbi-models": "^1.7.0",
-    "powerbi-visuals-api": "^4.2.0",
+    "powerbi-visuals-api": "~5.4.0",
     "powerbi-visuals-tools": "3.1.15",
```
Comment (zBritva) [inline package.json:61]: Update tools too

**Review by zBritva — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (SanderGi) [reply package.json]: Done [b8b0332](https://github.com/microsoft/powerbi-visuals-sunburst/pull/92/commits/b8b0332c91863f41bf54e97f17c16a0e8ebb7bef)

**Comment by SanderGi** [comment]
Closing because it is contained in this PR: https://github.com/microsoft/powerbi-visuals-sunburst/pull/93

**Pull request closed without merging**